src/pyfaktory/client.py

```diff
@@ -186,6 +186,7 @@ def disconnect(self):
         self.logger.info("Disconnecting...")
         self.sock.close()
         self._set_state(State.DISCONNECTED)
+        raise Exception("Disconnecting...")
```

Comment:
Why is it relevant to raise an Exception here?

Reply:
Outer can try catch this reason. Use this can manually handle logs and maintain the service.

Reply:
But disconnection can be requested, the exception must be raised only if this happens unexpectedly

Reply:
1.when use faktory dashboard ui stop this consume client ,deploy clinet server can receive  
2. when client self except  and disconnect  can record log

Reply:
Disconnection is logged, and state can be checked with `state` attribute, I think that we don't need to raise an Exception here.

Reply:
i think  throw an exception directly maybe better, if program exist self reconnect can easy to recod and restart and  try catch

Reply:
ok